.circleci/integration-test/10-check-whether-to-run-job.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+dockerized=$1
+image_family=$2
+trigger_source=$3
+branch=$4
+tag=$5
+
+log() { echo "$*" >&2; }
+
+if [[ -f "${WORKSPACE_ROOT}/pr-metadata/labels/skip-integration-tests" ]]; then
+    log "Skipping job with skip-integration-tests label." >&2
+    echo "false"
```

Comment:
Why are we using `echo` to output the result of the script and not using the exit code directly?